prow/config.yaml

```diff
@@ -7180,6 +7180,28 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  - name: pull-tf-minigo-cc
+    agent: kubernetes
+    always_run: true
+    skip_report: true
+    #context: pull-tf-minigo-cc
+    branches:
+    - master
+    rerun_command: "/test pull-tf-minigo-cc"
+    trigger: "/test pull-tf-minigo-cc"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/tensor-go/cc-base:latest
+        command:
+        - "/bin/sh -c \"cp -r /app/cc/tensorflow ./cc/tensorflow && bazel test cc:all --compilation_mode=dbg --define=board_size=9\""
+        #- bazel test cc:all --compilation_mode=dbg --define=board_size=19
```

Comment:
Should these commands be run sequentially as one job or should we run them as two jobs? If they take more than a couple minutes I would suggest using two jobs.

I'll enable the second test (or make another job for it) once we have something working.

Reply:
the tests take very little time -- a few seconds for the tests to run -- it's all bazel build and startup overhead.  Having it be two calls in sequence will save that overhead.